games-strategy/colobot-data/colobot-data-0.1.11.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils cmake-utils
+
+DESCRIPTION="Data package for colobot (Colonize with Bots)"
+HOMEPAGE="https://colobot.info/"
+
+SRC_URI="
+	https://github.com/colobot/colobot-data/archive/colobot-gold-${PV}-alpha.zip -> ${P}.zip
+	music_ogg? ( https://colobot.info/files/music/colobot-music_ogg_${PV}-alpha.tar.gz -> ${P}-music-ogg.tar.gz )
+	music_flac_convert? ( https://colobot.info/files/music/colobot-music_flac_${PV}-alpha.tar.gz -> ${P}-music-flac.tar.gz )
+"
+KEYWORDS="~amd64"
+DEPEND="music_flac_convert? ( media-sound/vorbis-tools )"
```

Comment:
variabled order is broken, why should `DEPEND` be here

Reply:
It converts flac music to ogg format with oggenc from vorbis-tools